pkg/operator/ceph/object/user/controller.go

```diff
@@ -151,7 +155,62 @@ func (c *ObjectStoreUserController) createUser(context *clusterd.Context, u *cep
 		DisplayName: &displayName,
 	}
 	objContext := object.NewContext(context, u.Spec.Store, u.Namespace)
-
+	// Wait with exponential backoff for the objectstore to be created
+	//
+	// ExponentialBackoff repeats a condition check with exponential backoff
+	//
+	// It checks the condition up to `Steps` times, increasing the wait by multiplying
+	// the previous `Duration` by `Factor`
+	//
+	// If `Jitter` is greater than zero, a random amount of each duration is added
+	// Jitter is applied after `Cap`
+	//
+	// If the condition never returns true, ErrWaitTimeout is returned. All errors
+	// except IsNotFound terminates immediately
+	err := wait.ExponentialBackoff(wait.Backoff{
```

Comment:
- I thought Steps is the number of retries it will increase the backoff period. So if we had `Steps: 3`, the settings would currently retry after 15, 30, 60, 60, 60, ... until it hits the Cap of 5 minutes. 
- For the comment, could you summarize the behavior? For example, if we just say something like "Will retry several times using exponential backoff, starting after 15 seconds and retrying several times until the timeout of 5 minutes is reached."

Reply:
+ Based on your assumption, it would no longer be an exponential backoff at one point.
+ Summaried with Example on it's behavior. This should be enable to backtrace later on, if required.

Reply:
The documentation for `struct wait.Backoff` does say something different than the documentation for `func wait.ExponenetialBackoff`. We should document this here since it has confused both Travis and myself.